final_project/dags/sql/customers_transfer_from_data_lake_raw_to_bronze.sql

```diff
@@ -0,0 +1,24 @@
+MERGE `de2022-stanislav-rezen.bronze.customers` main
+    USING (
+        select distinct * from customers_csv) stage
```

Comment:
Actually, this layer should be designed to match raw data as much as possible. So no deduplication should happen here. All data manipulation should be performed when we move the data from this layer to Silver.

Reply:
Got it!